src/server/services/financiele-hulp.ts

```diff
@@ -0,0 +1,105 @@
+import { Chapters } from '../../universal/config';
+import { apiDependencyError, apiSuccesResult } from '../../universal/helpers/api';
+import { MyNotification } from '../../universal/types';
+import { getApiConfig } from '../config';
+import { requestData } from '../helpers';
+
+interface NotificationTrigger {
+  datePublished: string;
+  url: string;
+}
+
+interface KrefiaDeepLink {
+  title: string;
+  url: string;
+}
+
+interface KrefiaDeepLinks {
+  budgetbeheer: KrefiaDeepLink | null;
+  lening: KrefiaDeepLink | null;
+  schuldhulp: KrefiaDeepLink | null;
+}
+
+interface NotificationTriggers {
+  fibu: NotificationTrigger | null;
+  krediet: NotificationTrigger | null;
+}
+
+export interface FinancieleHulp {
+  notificationTriggers: NotificationsTriggers;
+  deepLinks: KrefiaDeepLinks;
+}
+
+function createNotification(
+  message: NotificationTrigger,
+  type: string
+): MyNotification {
+  const isFibu = type === 'fibu';
+  return {
+    id: `financiele-hulp-${type}-notification`,
+    datePublished: message.datePublished,
+    title: isFibu ? 'Bericht FiBu' : `Bericht Kredietbank`,
+    chapter: Chapters.FINANCIELE_HULP,
+    description: isFibu
+      ? 'Er staan ongelezen berichten voor u klaar van FiBu'
+      : 'Er staan ongelezen berichten voor u klaar van de kredietbank',
+    link: { to: message.url, title: 'Bekijk uw bericht', target: '_blank' },
```

Comment:
```suggestion
    link: { to: message.url, title: 'Bekijk uw bericht' },
```

`_blank` gebruiken wij niet.